thamos/lib.py

```diff
@@ -379,7 +380,8 @@ def advise(
     limit: int = None,
     count: int = 1,
     debug: bool = False,
-    origin: str = None,
+    origin: typing.Optional[str] = None,
+    timeout: typing.Optional[int] = None,
```

Comment:
Thanks @fridex , can we extend also `advise_using_config` and other methods using `advise` method internally to be able to set `timeout`?